docs/grammar/SolidityLexer.g4

```diff
@@ -29,7 +29,7 @@ Do: 'do';
 Else: 'else';
 Emit: 'emit';
 Enum: 'enum';
-Error: 'error'; // not a real keyword
+Error: 'error';
 Revert: 'revert'; // not a real keyword
```

Comment:
Should we also make revert a keyword? Then we lose the revert function and it is impossible to have code that can be compiled with 0.8.0 and 0.9.0.

Reply:
@chriseth I think revert should stay as a regular symbol -for now-, as it is used like a function. OTOH i think it would make sense to make the language less prone to accidental errors (people misunderstanding code when they see the name `revert(...)`). But that could then be done in a later PR or even the next major release after that.

Reply:
For me the question would be whether we want to remove the `revert()` function. If we do, then I'd make it a keyword. If we don't then I guess it's not an option anyway.